modin/pandas/base.py

```diff
@@ -2073,6 +2073,8 @@ def resample(
         origin: Union[str, TimestampConvertibleTypes] = "start_day",
         offset: Optional[TimedeltaConvertibleTypes] = None,
     ):
+        from .resampler import Resampler
```

Comment:
Why are `Resampler`, `Window`, `Rolling` objects imported inside methods and not at the module level?

Reply:
Rebased. Thank you!